spring-integration-redis/src/main/java/org/springframework/integration/redis/inbound/RedisInboundChannelAdapter.java

```diff
@@ -78,6 +82,12 @@ public void setMessageConverter(MessageConverter messageConverter) {
 		this.messageConverter = messageConverter;
 	}
 
+	/**
+	 * Specify an {@link Executor} used for running the message listeners when messages are received.
+	 * @param taskExecutor the Executor to use for listener container.
+	 * @since 4.3.13
```

Comment:
Does this __really__ need to be back-ported? There's a work around (header enricher).

Reply:
Well, that's just a JavaDocs from the previous commit.

We indeed provide this `MESSAGE_SOURCE` feature only for `5.0`.

At the same time I'm not sure how you are going to workaround this header when we already step out of this `RedisInboundChannelAdapter`